source/sass/global.scss

```diff
@@ -68,6 +68,44 @@ hr {
       }
     }
   }
+
+  @at-root #view-home & {
+    $banner-height-mobile: 390px;
+    $banner-height: 480px;
+
+    height: $banner-height-mobile;
+
+    @media screen and (min-width: $bp-md) {
+      height: $banner-height;
+    }
+
+    .banner {
+      height: $banner-height-mobile;
+
+      @media screen and (min-width: $bp-md) {
+        height: $banner-height;
+      }
+
+      img {
+        height: 100%;
+        object-fit: cover;
```

Comment:
Do we need to support IE11? caniuse.com [says it's not supported](https://caniuse.com/#search=object-fit) but if IE11 isn't a concern, please ignore this comment.

Reply:
Thanks for flagging this. I'll file a ticket for this since there are already a few `object-fit` usages in our codebase. Let's ignore this for now as only 1.2% of our site visitors are on IE.

Reply:
ticket filed! https://github.com/mozilla/foundation.mozilla.org/issues/5040